docs/manual/upgrades.md

```diff
@@ -32,6 +32,18 @@ In the main container the new binaries will be present in `/var/dynamic-conf/bin
 The `UpdatePodConfig` subreconciler will ensure that all Pods have the new configuration present, this configuration ensures that the `fdbmonitor` will restart processes with the new binary present in the shared volume.
 Once all Pods have the new configuration and binaries present the operator can move to the next phase.
 
+A potentional blocker for moving towards the next Phase can be an issue with a subset of the Pods e.g. if the sidecar is unreachable.
+If the sidecar is unreachable the operator is not able to confirm that the new configuration and binary is in place.
+All Process Groups affected by such an issue can be discovered with the `SidecarUnreachable` condition.
```

Comment:
If any of the process groups are in "SidecarUnreachable" condition do we still advance to the Restart phase?

Reply:
No, this will basically block the upgrade until the issue is resolved. I can add an additional paragraph to make that more precise.

Reply:
Ok, that's what I remember.